python/lsst/meas/extensions/convolved/convolved.py

```diff
@@ -478,7 +486,7 @@ def measureAperture(self, measRecord, exposure, aperturePhot):
         """
         try:
             aperturePhot.measure(measRecord, exposure)
-        except:
+        except:  # noqa E722: want to catch any and all errors
```

Comment:
Still better to do `except Exception`, so you don't block e.g. `KeyboardInterrupt`, I think.